MPI/isx.c

```diff
@@ -612,11 +612,6 @@ static void log_times(char * log_file)
 {
   FILE * fp = NULL;
 
-  for(int i = 0; i < TIMER_NTIMERS; ++i){
```

Comment:
It might be better to leave a comment here why we removed this.

Reply:
@kholland-intel and @wrrobin - did this get resolved?

Would it suffice to include [this comment in `SHMEM/isx.c`](https://github.com/kholland-intel/ISx/blob/kholland-intel/timer_data/SHMEM/isx.c#L656) in the `MPI/isx.c`?

Reply:
Thanks @davidozog. This is ok, but I was also thinking whether we should leave a comment about why we kept `gather_rank_times` only and removed the counts. Let me know if this makes sense.

Reply:
Oh, I see.  Yeah, it makes sense - I think `gather_rank_counts` is being removed simply because it's not actually used nor included in the output log file.  It's at least in the git history... but maybe a breadcrumb comment would help revive it the future if need be.

Reply:
Thanks for the clarification @davidozog. Either way, I am ok then.